packages/lexical-editor/src/nodes/webinyNodes.ts

```diff
@@ -32,5 +33,12 @@ export const WebinyNodes: ReadonlyArray<
     OverflowNode,
     MarkNode,
     FontColorNode,
-    TypographyElementNode
+    TypographyElementNode,
+    BaseParagraphNode,
+    {
```

Comment:
Why do we need this replacement? Can we add a comment here?

Reply:
yes, I will comment, and respond here, for the custom implementation of inserting the theme styles. For rendering and editing updates.